App/src/main/java/cc/softwarefactory/lokki/android/PreferencesFragment.java

```diff
@@ -0,0 +1,45 @@
+package cc.softwarefactory.lokki.android;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceFragment;
+import android.util.Log;
+
+import cc.softwarefactory.lokki.android.utils.PreferenceUtils;
+import cc.softwarefactory.lokki.android.utils.Utils;
+
+public class PreferencesFragment extends PreferenceFragment implements SharedPreferences.OnSharedPreferenceChangeListener {
```

Comment:
I think this class should be named for instance `SettingsFragment` as the current naming creates confusion with the class it is extending. It almost overrides it.